scalar2/oom_handler.go

```diff
@@ -148,7 +148,7 @@ func (p *OOMKillsProcessor) handlePod(pod corev1.Pod) error {
 		return nil
 	}
 
-	parents, err := kuber.GetParents(
+	parents, _ := kuber.GetParents(
```

Comment:
why this error is escaped? what would happen if the `kuber.RootParet(parents)` on line 162, if the parents where empty obj or null? would if fail? if it would then you need to handle the error.

@mohamedveron

Reply:
we will remove this handler